.github/workflows/azure-container-webapp.yml

```diff
@@ -38,44 +44,44 @@ permissions:
   contents: read
 
 jobs:
-  build:
+  build-dev:
     runs-on: ubuntu-latest
+    environment: dev
 
     steps:
       - uses: actions/checkout@v3
 
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
-
-      - name: Log in to GitHub container registry
-        uses: docker/login-action@v1.10.0
+        uses: docker/setup-buildx-action@v2
         with:
-          registry: ghcr.io
-          username: ${{ github.actor }}
-          password: ${{ github.token }}
+          context: ../../src/
+          file: ../../src/Dockerfile
```

Comment:
What directory is this based out of?

Reply:
resolve and handle in separate conversation/branch